src/libraries/Common/src/Interop/Unix/System.Native/Interop.GNameUName.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using System.Buffers;
+using System.Text;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        /// <summary>
+        /// Gets the group name associated to the specified group ID.
+        /// </summary>
+        /// <param name="gid">The group ID.</param>
+        /// <returns>On success, return a string with the group name. On failure, throws an IOException.</returns>
+        internal static string GetGName(uint gid)
+        {
+            string result = GetGNameInternal(gid);
+            return (result == null) ? throw GetIOException(GetLastErrorInfo()) : result;
```

Comment:
This could just be:

return GetGNameInternal(gid) ?? throw GetIOException(GetLastErrorInfo());